packages/SidePanel/src/components/Dialog/Dialog.styles.js

```diff
@@ -1,12 +1,12 @@
-import { spaceSm, spacer } from "@paprika/stylers/lib/helpers";
```

Comment:
@nahumzs is this form of importing stylers problematic?

Reply:
https://github.com/acl-services/paprika/pull/213/commits/b833c0614d2ff4cb4b3068eb6a676c79c0c984f8

I had to modified in one instance in the ListBoxBrowser to make it pass, maybe for some reason is breaking your branch? worth trying ...

Reply:
shouldn't be a problem ...